packages/roosterjs-editor-api/lib/format/execFormatWithUndo.ts

```diff
@@ -10,12 +10,19 @@ import { EditorPoint, NodeType } from 'roosterjs-editor-types';
  */
 export type Formatter = (startPoint: EditorPoint, endPoint: EditorPoint) => Node | void | any;
 
-// Exec format with undo
+/**
+ * Execute format with undo
```

Comment:
Need to mention, it tries to add undo snapshot at begin and end of the function. And duplicated snapshot will be only added once.

Reply:
Added